packages/storybook/src/nimble/tree-view/tree-view.stories.ts

```diff
@@ -135,9 +167,29 @@ export const anchorTreeItem: StoryObj<AnchorItemArgs> = {
 export const multipleTreeItems: StoryObj<TreeArgs> = {
     argTypes: {
         selectionMode: {
+            name: 'selection-mode',
             options: Object.values(TreeViewSelectionMode),
-            control: { type: 'radio' }
-        }
+            control: { type: 'radio' },
+            description: selectionModeDescription,
+            table: { category: apiCategory.attributes }
+
+        },
+        options: {
+            name: 'default',
+            description:
+                `One or more \`${treeItemTag}\` or \`${anchorTreeItemTag}\` elements which populate the tree. Nest items to establish tree hierarchy.`,
+            table: { category: apiCategory.slots }
+        },
+        expandedChange: {
+            name: 'expanded-change',
+            description: 'Event that emits when an item is expanded or collapsed.',
```

Comment:
nit: Should use consistent wording, i.e. "Event that emits" vs "Event emitted" vs "Emitted when" vs "Fires when". Seems like we mostly use "Event emitted".
```suggestion
            description: 'Event emitted when an item is expanded or collapsed.',
```